### DISCUSSION TITLE
Read-me-stat not updating

### DISCUSSION THREAD

**Discussion by aimedivin** [anuraghazra/github-readme-stats#3010] [Q&A] [answered]
My read me stat is delaying in updating when anything changes or if there is a commit, kindly assist.

**Comment by qwerty541** [comment, marked as answer]
Hey, @aimedivin! Stats card update time depends from `cache_seconds` query option. By default it was set to 14400 seconds ( 4 hours ). You can set custom value from 14400 to 86400 seconds. See: https://github.com/anuraghazra/github-readme-stats#common-options. You are also able to [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) and set `CACHE_SECONDS` environment variable, this makes it possible to set a cache seconds without upper and lower level restrictions.

**Comment by aimedivin** [comment]
Hey, Thanks for sharing, means that it could be updated every 4hours , so
can this cache-seconds be set under the default one ?

thank you

**Reply by rickstaa** [reply]
> Hey, @aimedivin! Stats card update time depends from `cache_seconds` query option. By default it was set to 14400 seconds ( 4 hours ). You can set custom value from 14400 to 86400 seconds. See: https://github.com/anuraghazra/github-readme-stats#common-options. You are also able to [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) and set `CACHE_SECONDS` environment variable, this makes it possible to set a cache seconds without upper and lower level restrictions.

Hey @aimedivin as stated by @qwerty541 you can set it lower by [deploying your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) and set `CACHE_SECONDS` environment variable in Vercel.

**Comment by aimedivin** [comment]
Thank you sir for the tip!
